Sources/AWSSDKSwiftCore/Message/Body.swift

```diff
@@ -77,4 +77,41 @@ extension Body {
             return nil
         }
     }
+
+    /// return as bytebuffer
+    public func asByteBuffer() -> ByteBuffer? {
+        switch self {
+        case .text(let text):
+            var buffer = ByteBufferAllocator().buffer(capacity: text.utf8.count)
+            buffer.writeString(text)
+            return buffer
+
+        case .buffer(let data):
+            var buffer = ByteBufferAllocator().buffer(capacity: data.count)
+            buffer.writeBytes(data)
+            return buffer
+
+        case .json(let data):
+            if data.isEmpty {
```

Comment:
```
guard !data.isEmpty else {
  return nil
}

// continue with no indent.
```

Reply:
Not very sure why this code is even in there. I can't see a reason how someone could initialize json with an empty body. Also the test is only there for json. I have another branch where I am working on Body can we look into this then when I put it up (next few days)

Reply:
no fix for now.